src/changed-projects.js

```diff
@@ -2,8 +2,8 @@ const { join } = require('path')
 const { getPackagesPaths, getAllChanges } = require('./libs/utils')
 
 module.exports = async (options = {}) => {
-  const rushRootPath = join(process.cwd())
-  const rushChangePath = join(process.cwd(), 'common', 'changes')
+  const rushRootPath = join(process.cwd(), options.workingDirectory || '.')
```

Comment:
Is the `|| '.'` part required if it's provided by the default value?

Reply:
This conditioned is required for jest unit testing, because jest will directly use this function and `options.workingDirectory` will be `undefined`